spp_import_dci_api/models/individual.py

```diff
@@ -0,0 +1,46 @@
+from odoo import api, fields, models
+
+
+class OpenSPPIndividual(models.Model):
+    _inherit = "res.partner"
+
+    ind_is_imported_from_crvs = fields.Boolean(
+        "Imported from CRVS", compute="_compute_ind_is_imported_from_crvs"
+    )
+
+    crvs_import_ids = fields.One2many(
+        "spp.crvs.imported.individuals", "individual_id", "CRVS Import"
+    )
+
+    is_mother = fields.Boolean("A Mother", compute="_compute_is_mother", store=True)
+
+    location_id = fields.Many2one("spp.crvs.location")
+
+    @api.depends("crvs_import_ids")
+    def _compute_ind_is_imported_from_crvs(self):
+        for rec in self:
+            if rec.crvs_import_ids:
+                rec.ind_is_imported_from_crvs = True
+            else:
+                rec.ind_is_imported_from_crvs = False
+
+    @api.depends("individual_membership_ids")
+    def _compute_is_mother(self):
```

Comment:
@reichie020212 With this, do we want to block the changing of group membership kind for objects imported from CRVS? Should we also check for this? Or check that the gender is also female? Otherwise, I believe a male can become a mother.